lib/js-api/src/api/content-rest-api/api/preferences.api.ts

```diff
@@ -84,4 +84,25 @@ export class PreferencesApi extends BaseApi {
             queryParams
         });
     }
+
+    updatePreference(personId: string, preferenceName: string, preferenceValue: string): Promise<PreferenceEntry> {
```

Comment:
should we have some unit tests for this? I see for other APIs we have unit tests like for tag api file.

Reply:
I wonder if we actually need them, to be honest they're quite outdated, a lot of APIs are not covered and maybe for the future we could think of a better solution for this but for now I would not add new unit tests there

Reply:
ok, sounds good